lib/opal/nodes/args/kwrestarg.rb

```diff
@@ -0,0 +1,40 @@
+require 'opal/nodes/args/initialize_kwargs'
+
+module Opal
+  module Nodes
+    # A node responsible for extracting a
+    # keyword splat argument
+    #
+    # def m(**kwrest)
+    # def m(**)
+    #
+    class KwrestArgNode < InitializeKwargsNode
+      handle :kwrestarg
+
+      def compile
+        super
+
+        kwrestarg_name = @sexp[1]
+        extract_code = "Opal.kwrestargs($kwargs, #{used_kwargs});"
+
+        if kwrestarg_name
+          var_name = variable(kwrestarg_name.to_sym)
+          add_temp var_name
+
+          line "#{var_name} = #{extract_code}"
+        else
+          # kwrestarg can be blank def m(**) end
+          line extract_code
```

Comment:
Is this really necessary? couldn't it just be skipped? (sorry if it's a dumb question…)